connectors/sources/sharepoint_online.py

```diff
@@ -1005,6 +1022,15 @@ def get_default_configuration(cls):
                 "type": "bool",
                 "value": False,
             },
+            "fetch_drive_item_permissions": {
+                "depends_on": [{"field": "use_document_level_security", "value": True}],
+                "display": "toggle",
+                "label": "Fetch drive item permissions",
+                "order": 9,
+                "tooltip": "Enable this option to fetch drive item specific permissions. Note that this setting can potentially increase sync time.",
+                "type": "bool",
+                "value": True,
```

Comment:
This is expected to be `True` by default, and I don't see `8.9` label in the PR, is migration needed if customers upgrade from `8.9` to `8.10`?

Reply:
Yes, I would expect that we need one.

Reply:
Good call, Chenhui. Tim, can you create and link an issue so we don't forget?

Reply:
Created it last week: https://github.com/elastic/enterprise-search-team/issues/5181